### ISSUE TITLE
JSON export for all tables in SavedVariables file

### ISSUE THREAD

**Issue by Fjarnskaggl** [Alumian/CEPGP-Retail#135]
It would be tremendously useful to be able to export all data (tables) stored in the SavedVariables file for CEPGP to JSON.

**Comment by sadsfae** [comment]
Howdy, in trying to figure out how to export item and point values came across the `SavedVariables\CEPGP.lua` file.  Is it possible to also have this exported to CSV format?

FWIW I am trying to use this with EQDKP-Plus EPGP-log-importer plugin to auto-populate the "items" area:

https://github.com/EQdkpPlus/plugin-epgpimport

Is this this file the only place items/points/per-raid data is stored?

**Comment by Fjarnskaggl** [comment]
Correct, that is the only file that the data you're looking for is stored.
